team-fingerprints-web/src/components/Chart/Chart.tsx

```diff
@@ -60,7 +71,7 @@ const Chart: FC<IProps> = ({
   useEffect(() => {
     setWidth(ref.current?.clientWidth || 0);
     setHeight(ref.current?.clientHeight || 0);
-  }, [screenWidth]);
+  }, [screenWidth, categoriesToFilter?.[0]]); //Important! need to be here to recompute height when new filters applied
```

Comment:
Is targeting first array element necessary here? 
Might cause some issues in the future if the picker will be changed to handle multiple categories and someone forgets to change this dependency
```suggestion
  }, [screenWidth, categoriesToFilter]); //Important! need to be here to recompute height when new filters applied
```